### PR TITLE
fix #299654: Crash on startup with Qt 5.14

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
resolves https://musescore.org/en/node/299654, code taken from patch provided there

### CODE REVIEW

**Comment by Howard-C** [comment]
No hash-tag in "fix 299654"?

File: mscore/workspace.cpp [hunk 1]
```diff
@@ -823,9 +823,9 @@ void Workspace::read(XmlReader& e)
                   QMenuBar* mb = mscore->menuBar();
                   const QObjectList menus(mb->children()); // need a copy
                   for (QObject* m : menus) {
-                        if (qobject_cast<QMenu*>(m)) {
-                              m->setParent(nullptr);
-                              m->deleteLater();
+                        if (QMenu* menu = qobject_cast<QMenu*>(m)) {
```
Comment (vpereverzev) [inline mscore/workspace.cpp]: What exactly the reason of crash?

And why QMenuBar::clear() isn't enough? Did you tested it without 827 and 828?

Please, move local variable from the if condition.

Comment (Jojo-Schmitz) [reply mscore/workspace.cpp]: I've just turned the patch provided in  https://musescore.org/en/node/299654 into a PR.
Changes to it should better discussed there I guess?

Comment (vpereverzev) [reply mscore/workspace.cpp]: Yeah, sorry, didn't noticed that he attached the patch.

**Pull request merged**